### DISCUSSION TITLE
Welcome to kemai Discussions!

### DISCUSSION THREAD

**Discussion by AlexandrePTJ** [AlexandrePTJ/kemai#92] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kfleten** [comment]
My name is Kjetil Fleten, and I have used Kemai togheter with Kimai for some years. I love opensource, and have been running Linux and opensource software since 2001. I use Kemai for timetracking in my business and as a freelancer.
